src/main/java/org/fundacionjala/coding/jorge/DeodorantEvaporator.java

```diff
@@ -0,0 +1,30 @@
+package org.fundacionjala.coding.jorge;
+
+/**
+ *
+ */
+public class DeodorantEvaporator {
+    /**
+     *
+     */
+    protected DeodorantEvaporator() {
+    }
+
+    /**
+     * @param content type double.
+     * @param evapPerDay type double.
+     * @param threshold    type double.
+     * @return int vowels count.
+     */
+    public static int evaporator(double content, double evapPerDay, double threshold) {
```

Comment:
since you are instantiating in your unit tests
the static reserved word is no longer necessary.